src/prefect/variables.py

```diff
@@ -69,27 +66,23 @@ def my_flow():
                 variable=VariableRequest(**var_dict)
             )
 
-        return variable if as_object else variable.value
+        return variable
 
     @classmethod
     @sync_compatible
     async def get(
         cls,
         name: str,
         default: StrictVariableValue = None,
-        as_object: bool = False,
-    ) -> Union[StrictVariableValue, VariableResponse]:
+    ) -> str:
```

Comment:
Looks like we might want to keep this `StrictVariableValue` since a variable can be any JSON-serializable value.
```suggestion
    ) -> StrictVariableValue:
```

Reply:
+1